xml/System.Net.Security/SslClientAuthenticationOptions.xml

```diff
@@ -179,9 +199,14 @@
         <ReturnType>System.Security.Cryptography.X509Certificates.X509CertificateCollection</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>List of the client certificates offered to the server.</summary>
```

Comment:
We should make sure to be clear we will pick just one from the collection and it must have private key. 
There are some weird rules hard to describe and there were cases when we did not pick anything from the collection  - sometimes even if there is exactly one certificate.